tests/test_main.py

```diff
@@ -0,0 +1,8 @@
+import subprocess
+
+
+def test_main_script():
+    result = subprocess.run(["python", "src/main.py", "Hello, World!"],
```

Comment:
The default model used is quite large for testing, please use a smaller model that requires less resources so it will be fit for testing env.